versioned_docs/version-v4.0.0/node/frequently-asked-questions.md

```
@@ -0,0 +1,461 @@
+
+:::info important
+
+The Cover Traffic threshold rewards cap limitations mentioned above do not apply to rewards received from Cover Traffic. If you stake up to **75,000 wxHOPR** per node and the rewards you receive increase your stake, you will still **receive the full APR**!
+
+:::
+</details>
+
+<details>
+<summary>
+
+### What is my expected APR?
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)